src/components/goalDetail/GoalTodoCreateModal.tsx

```
@@ -0,0 +1,124 @@
+    .nonempty('제목을 입력해주세요')
+    .max(30, '제목은 30자 이하여야 합니다'),
+  goal: z.any().nullable().optional(),
+  date: z.union([z.date(), z.string()]).optional(),
+});
+
+type TodoFormValues = z.infer<typeof createTodoSchema>;
+
+type Props = {
+  goalId: string;
+  onClose: () => void;
+  onAdd: (task: string, date: string) => void;
```

ESLint

[error] 'task' is defined but never used. Allowed unused args must match /^_/u.

(no-unused-vars)